overlord/snapstate/snapstate.go

```diff
@@ -64,6 +64,9 @@ const (
 
 const (
 	DownloadAndChecksDoneEdge = state.TaskSetEdge("download-and-checks-done")
+	PrerequisitesEdge         = state.TaskSetEdge("prerequisites")
```

Comment:
compare with DownloadAndChecksDone ? these names feel a bit too internal,  it would be better to have higher-level names,  like BeginEdge?    BeforeHooksEdge ? etc?